git-workflow.md

```diff
@@ -7,6 +7,22 @@
 * Choose a merging strategy (cf. below) based on your context ;
 * Tags: Each release of Wazo must be tagged in applicable repositories. Tags must be PGP-signed.
 
+## Reviewing and validating pull request
+
+### How to do it?
+
+* set JIRA ticket to `in review`
+* assign to someone, self-assign or add _:pray:`please review`_
+* review, at least one, by someone who don't write the code
+  * fetch PR locally with 
+
+		git pull-request --download <PR-number>
+
+* Once PR is valid, "Approve" it (in review tab)
+* if valid add `mergeit` label to trigger zuul
```

Comment:
What is the difference to add the label `mergeit` when the ticket is ready to be reviewed VS when the ticket is reviewed? In all case, zuul will wait the approval of someone before merging, no?

Reply:
> ... zuul will wait the approval of someone before merging ...

Unfortunately, this is a bad implementation of zuul+github, on Gerrit you don't have such restriction once you put mergeit (called +Approved in gerrit) and tests passed, the change is merged.

This `mergeit` + `at least 1 approvals` doesn't looks configurable for now :(

Reply:
Oh good to know, thanks